game.js

```diff
@@ -328,6 +365,25 @@ class Game extends Screen {
 		}
 	}
 
+	returntoMenuAfterWin() {
+			
+			this.noGuessChar = '_'
+    		this.livesRemaining = 0
+			this.guess = []
+			this.wrongGuesses = []
+			this.score = 0
+			this.currentSpinOption = new SpinOption(100)
+			this.spinCount = 0 // The number of times to spin for points
+			this.spinResultSequence = 0
+			this.level = 0 // The current level
+			this.correctLetterIndices = []
+			this.incorrectGuessChar = null
+			this.pauseUntilMilliSecond = 0 // The # of ms since the program started to pause until
+			this.puzzleRevealCountdown = 0
+   	 		this.gotoNextLevel()
+			state = MAIN_SCREEN_STATE
```

Comment:
What is the point of this local variable `state`?

Reply:
It is to change the state to the main screen state which in the draw function shows the main screen. I want the code to work like this: 
1. Check if score >= target score
2. Bring up a win screen menu or a win screen animation (I was going to add buttons for return to menu and play again, but it seems simpler to just send the user back to the menu after some time like you suggested.)
3. Reset the game, so when you play again the score, levels, and all the important variables are reset.